shell/browser/webauthn/electron_authenticator_request_delegate.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Why is this code copyright the chomium authors, afaics this code does not match anything from the current Chromium source tree and is simply a minimal override implementation of `WebAuthenticationDelegate`

Reply:
It was originally based on the chrome version so I kept the copyright, but its really so different now it can have its own.
I've put in the github version, similar to other files in the codebase.